mailpoet/mailpoet
# Add more options for the WooCommerce coupon block in the email editor - MAILPOET-4762

## Description

Add coupon settings to the Email editor WooCommerce coupon block.

## Code review notes

Please start reviewing from https://github.com/mailpoet/mailpoet/commit/92b9d11fe952abd76f8d90d780464a9325948f36

## QA notes

You need to enable Coupon block experimental features here: `/wp-admin/admin.php?page=mailpoet-experimental`

* On the email editor
* Select the coupon block
* Click on the Usage restriction or Usage limits
* Add some options
* Send the newsletter
* Your newsletter should have the new coupon code. 
* You may verify the coupon options at `/wp-admin/edit.php?post_type=shop_coupon`, select the coupon code found in the newsletter

## Linked PRs

Dependent on https://github.com/mailpoet/mailpoet/pull/4636

## Linked tickets

[MAILPOET-4762](https://mailpoet.atlassian.net/browse/MAILPOET-4762)




[MAILPOET-4762]: https://mailpoet.atlassian.net/browse/MAILPOET-4762?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
